apps/bfd-data-fda/src/main/java/gov/cms/bfd/data/fda/lookup/FdaDrugCodeDisplayLookup.java

```diff
@@ -19,17 +19,11 @@
 public class FdaDrugCodeDisplayLookup {
   private static final Logger LOGGER = LoggerFactory.getLogger(FdaDrugCodeDisplayLookup.class);
 
-  /** A fake drug code used for testing. */
-  public static final String FAKE_DRUG_CODE = "00000-0000";
-
-  /** A fake drug code display that is associated with the FAKE_DRUG_CODE. */
-  public static final String FAKE_DRUG_CODE_DISPLAY = "Fake Diluent - WATER";
-
   /**
    * Stores a map from Drug Code (PRODUCTNDC) to Drug Code Display (SUBSTANCENAME) derived from the
    * downloaded NDC file.
    */
-  private Map<String, String> ndcProductHashMap = new HashMap<>();
+  public Map<String, String> ndcProductHashMap = new HashMap<>();
```

Comment:
I think I would prefer keeping this field private and still having a constructor that accepts the map as a parameter. Making this public breaks encapsulation.

Reply:
after some trials and errors to avoid adding the constructor back and keeping 'ndcProductHashMap' private, I have went on with it and added the constructor back :(.
Let me know if you maybe think of a better way of achieving this.

Reply:
Having the constructor that accepts the map is fine. That's just dependency injection. We wanted to get rid of the stuff that explicitly hardcodes test data in here and having the constructor still allows us to avoid doing that.